### ISSUE TITLE
RatingIndicator: add `required` and `accessibleNameRef`

### ISSUE THREAD

**Issue by Lukas742** [SAP/ui5-webcomponents#5894]
### **Feature Request Description**
For screen reader support we need to be able to mark every input-like component as required and also add a label to them. For this we would need the `RatingIndicator` to accept `required` and `accessibleNameRef`. 

### **Proposed Solution**
Add `required` and `accessibleNameRef` to the component's public properties/attributes.

### **Additional Context**
We need this for our [FilterBar](https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-filterbar--default-story) implementation. There it's possible to set a label for a filter and also make it mandatory (required). Per default screen readers don't read the label on top of the filter (for that we need `accessibleNameRef`) and if the filter is required, the input should be required as well (for that we need `required`).

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

A clear and concise description of the impact/urgency of the required feature.

### **Stakeholder Info** (if applicable)

- Organization: UI5 Web Components for React

**Comment by gmkv** [comment]
Hi @SAP/ui5-webcomponents-topic-rl 
This is a feature request to add `required` and `accessibleNameRef` to the RatingIndicator.
Note that there isn't a `required` equivalent in the corresponding OpenUI5 control.
Would you have a look?

**Comment by nikolay-kolarov** [comment]
Hello,
We have contacted the UX designers, as the enablement of this might generate accessibility issues and it is unclear if by the current design the RatingIndicator is expected to be labelled and required.
Best Regards,
Nikolay
